src/Apps/Sell/Routes/ArtistNotEligibleRoute.tsx

```diff
@@ -42,10 +72,8 @@ export const ArtistNotEligibleRoute: React.FC<ArtistNotEligibleRouteProps> = pro
 
           <Flex flexDirection="column" justifyContent="space-between" pb={2}>
             <Button
-              // @ts-ignore
-              as={RouterLink}
-              to="/collector-profile/my-collection/artworks/new"
               data-testid="add-to-collection"
+              onClick={onAddToMyCollection}
```

Comment:
Can we leave the `as={RouterLink}` in combination with the `onClick` so the button is a real link? Like this, it would be possible to do stuff like command+click to open the link in a new tab.

```suggestion
              // @ts-ignore
              as={RouterLink}
              to="/collector-profile/my-collection/artworks/new"
              onClick={onAddToMyCollection}
```

Reply:
Good call – I'll just need to prevent the default behavior on click in the `onClick` handler. 👍🏼